packages/react-router-dom/modules/Link.js

```diff
@@ -58,6 +58,10 @@ class Link extends React.Component {
   render() {
     const { replace, to, ...props } = this.props // eslint-disable-line no-unused-vars
 
+    if (to == null) {
+      return <a {...props} href={null} />
```

Comment:
Set to null to avoid mistype when you pass `href` prop instead of `to`. Link with non-empty `to` always generate it's own `href`, so null is required only for empty one.